scripts/package

```diff
@@ -14,7 +14,7 @@ REPO=${REPO:-rancher}
 
 cp ../bin/rancher-vxlan .
 
-IMAGE=${REPO}/vxlan:${TAG}
+IMAGE=${REPO}/rancher-vxlan:${TAG}
```

Comment:
We don't want the image to be `rancher/rancher-vxlan`, `rancher/vxlan` would be sufficient.